services/cloudapi/cloudapinic/nic.go

```diff
@@ -6,11 +6,77 @@ import (
 	"log"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
 	ionoscloud "github.com/ionos-cloud/sdk-go/v6"
 	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/services/cloudapi"
 	"github.com/ionos-cloud/terraform-provider-ionoscloud/v6/utils"
 )
 
+var FlowlogSchemaResource = &schema.Resource{
+	Schema: map[string]*schema.Schema{
+		"id": {
+			Type:        schema.TypeString,
+			Computed:    true,
+			Description: "The resource's unique identifier.",
+		},
+		"action": {
+			Type:        schema.TypeString,
+			Description: "Specifies the traffic direction pattern. Valid values: ACCEPTED, REJECTED, ALL. Immutable, forces re-recreation of the nic resource.",
+			Required:    true,
+			ForceNew:    true,
+		},
+		"bucket": {
+			Type:        schema.TypeString,
+			Description: "The S3 bucket name of an existing IONOS Cloud S3 bucket. Immutable, forces re-recreation of the nic resource.",
+			Required:    true,
+			ForceNew:    true,
+		},
+		"direction": {
+			Type:        schema.TypeString,
+			Description: "Specifies the traffic direction pattern. Valid values: INGRESS, EGRESS, BIDIRECTIONAL. Immutable, forces re-recreation of the nic resource.",
+			Required:    true,
+			ForceNew:    true,
+		},
+		"name": {
+			Type:             schema.TypeString,
+			Description:      "The resource name. Immutable, forces re-recreation of the nic resource.",
+			Required:         true,
+			ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotWhiteSpace),
+			ForceNew:         true,
```

Comment:
@cristiGuranIonos I'm just double checking that we intend to have `ForceNew` here for the `name` attribute since this is not happening for other resources.